superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -0,0 +1,161 @@
+/**
```

Comment:
Not a huge deal, but let's use `git mv` to keep the history of the file attached to it! You can fix it quickly with
```bash
mv superset-frontend/src/datasource/ChangeDatasourceModal.tsx /tmp
git checkout HEAD^ superset-frontend/src/datasource/ChangeDatasourceModal.tsx
git mv superset-frontend/src/datasource/ChangeDatasourceModal.jsx superset-frontend/src/datasource/ChangeDatasourceModal.tsx
mv /tmp/ChangeDatasourceModal.tsx superset-frontend/src/datasource/
```

Reply:
I couldn't get moving history working for git